src/components/forms/SubmittedFormsList.tsx

```diff
@@ -26,8 +26,13 @@ const SubmittedFormsList = ({
     (form: IFormR) => form.lifecycleState === LifeCycleState.Submitted
   );
   let content: JSX.Element | JSX.Element[];
+  const SUBMISSION_DATE: string = "submissionDate";
```

Comment:
Not sure if this is just a code style thing, but there's only one use of this constant - should we just hard code it on line 33?